man/wavpack.1

```diff
@@ -103,7 +103,8 @@ On by default in lossless mode and with the
 .Fl cc
 option.
 .It Fl d
-Delete source file if successful.
+delete source file if successful;
+.Sy use with caution!
```

Comment:
I wouldn't mark this up at all, the exclamation mark is imho enough :-)
Perhaps Em (emphasis) would be more suitable than Sy (symbolic).

Reply:
I'm happy to take it out if you feel strongly about it. But I believe that Sy (which the mdoc(7) page describes as for "importance or seriousness" and uses a warning as an example) is better than Em. Emphasis is used to _clarify_ a statement, which is _not_ the same as stressing its importance. Like I did there...  :smile_cat:

Besides, when I tried Em on my system I got a broken underline, which was ugly.

Reply:
OK, no strong feelings.